Source/WarpX.cpp

```diff
@@ -582,12 +587,18 @@ WarpX::ReadParameters ()
         pp_warpx.query("do_pml_j_damping", do_pml_j_damping);
         pp_warpx.query("do_pml_in_domain", do_pml_in_domain);
 
+        if (do_multi_J && do_pml)
+        {
+            amrex::Abort("Multi-J algorithm not implemented with PMLs");
+        }
```

Comment:
We could add one more warning: `do_multi_J` should be used with `fft_do_time_averaging` cc @RemiLehe :)

Reply:
By looking at `WarpX::OneStep_multiJ` in Source/Evolve/WarpXEvolve.cpp, I thought that the multi-J PIC loop would work even without time averaging. Isn't it the case, @RemiLehe?

Reply:
As discussed offline, I added a note in the documentation of the new input parameter `warpx.do_multi_J` (https://github.com/ECP-WarpX/WarpX/pull/1828/commits/4a0d065f28cebaad500cbdd727fc4bbf8c37dfb1), but we do not add a warning in the code, as it could be confusing to users who want to activate the multi-J algorithm for use cases that have nothing to do with NCI.